app/src/engineering/java/com/hedvig/app/DevelopmentActivity.kt

```diff
@@ -34,6 +43,14 @@ class DevelopmentActivity : AppCompatActivity(R.layout.activity_development) {
                             .putExtra(LoggedInActivity.EXTRA_IS_FROM_ONBOARDING, true)
                     )
                 },
+                DevelopmentScreenAdapter.DevelopmentScreenItem.Row("Logged in with whats new-screen") {
+                    unloadKoinModules(listOf(whatsNewModule))
```

Comment:
Move this to its own mock activity, where we can perform proper cleanup. As it stands, this would cause other mock activities to be affected.

We can do this after the fact, however, as we want to merge this asap.